src/main/scala/io/qbeast/spark/QbeastTable.scala

```diff
@@ -83,10 +82,12 @@ class QbeastTable private (
     }
   }
 
-  def optimize(options: Map[String, String] = Map.empty): Unit = {
-    if (!isStaging(latestRevisionAvailableID)) {
-      optimize(latestRevisionAvailableID, options)
-    }
+  def optimize(options: Map[String, String]): Unit = {
+    optimize(latestRevisionAvailableID, options)
+  }
+
```

Comment:
We should add a method when the user wants to optimize a RevisionID with not options:
```scala
def optimize(revisionID: RevisionID): Unit = {
  optimize(revisionID, Map.empty[String, String])
}
```

Then, another method for a sequence of input file paths without options:
```scala
def optimize(files: Seq[String]): Unit =
  optimize(files, Map.empty[String, String])
```

Reply:
done